### DISCUSSION TITLE
FORM表单组件，在验证后为什么无法执行finally？

### DISCUSSION THREAD

**Discussion by ismeljm** [tusen-ai/naive-ui#2033] [General]
const doSubmitForm = (e:Event) => {
      e.preventDefault()
      form.value.validate((errors:Array<FormValidationError>) => {
        formState.checkLoading = true
        if (errors) {
          message.error('账号或密码错误')
        } else {
          message.success('验证成功')
        }
      }).finally(()=>{
          console.log('没打印出来')
          formState.checkLoading = false
      })
    }

**Comment by 07akioni** [comment]
这我还真不知道，成功或者失败打出来了么？

**Reply by ismeljm** [reply]
form.value.validate()不是应该返回一个Promise吗？而调用Promise的finally方法时，里面的console.log('没打印出来')和formState.checkLoading = false没有执行，不知道为什么？

**Reply by 07akioni** [reply]
@Talljack 关注一下

**Reply by ismeljm** [reply]
OK

**Reply by 07akioni** [reply]
@Talljack 有复现吗？

**Reply by Talljack** [reply]
是个bug，下一版会修复 @ismeljm

**Reply by ismeljm** [reply]
@Talljack 期待新版本^_^
